example-gto/code/mlem/.mlem/config.yaml

```diff
@@ -0,0 +1,6 @@
+#core:
+#  state:
+#    uri: s3://gto-mlem-example/state
```

Comment:
If we will encourage users to clone the repo to play with it, this can be a show-stopper since it requires to have active AWS subscription. I will investigate, maybe we can use some github infra (gists? wf artifacts?) to store state

Reply:
@mike0sv, do you have any updates re this? Also, could you please populate the AWS secrets for https://github.com/iterative/example-gto?

Reply:
Maybe heroku app that runs mongodb or minio could work?

Reply:
I'm using s3 now with your creds, think we can improve this after release. Created an issue for this https://github.com/iterative/example-repos-dev/issues/134